lib/features/drawing_detail/drawing_detail_bloc.dart

```diff
@@ -129,4 +132,43 @@ class DrawingDetailBloc extends Bloc<DrawingDetailEvent, DrawingDetailState> {
     }
   }
 
+  Future<void> _addRecentDrawing(AddRecentDrawingEvent event, Emitter<DrawingDetailState> emit) async {
+    // User
+    User? currentUser = FirebaseAuth.instance.currentUser;
+    if (currentUser == null) {
+      return emit(DrawingDetailStateError(errorMessage: "User doesn't exist"));
+    }
+
+    String userId = currentUser.uid;
+
+    // Adding recent logic
+    FirebaseFirestore.instance.useFirestoreEmulator('192.168.1.18', 8080);

```

Comment:
Using local emulator or real backend should be transparent. Doesn't make sense to target local emulator. The configuration is done in the `.env`

Reply:
Yes I was testing this as data from firebase is not loaded as it is, forgot to remove after that